fgspectra/cross.py

```diff
@@ -42,17 +34,36 @@ class FactorizedCrossSpectrum(CrossSpectrum):
     broadcast-compatible.
     """
 
-    def __init__(self, sed, cl):
+    def __init__(self, sed, cl, **kwargs):
```

Comment:
`**kwargs` adds the ability to define the defaults at construction. This choice makes sense only if we use it consistently in all the Models we'll implement...and, of course, any choice is a limitation

Reply:
People want this, and I'm glad we're doing it using kwargs since that's also how users will call the Model.